skglm/penalties/__init__.py

```diff
@@ -1,9 +1,14 @@
-from .base import BasePenalty  # noqa F401
```

Comment:
sorry, I may have been unclear: there is no particular reason to change this here. It makes the diff longer than it should

The PR is not concerned with penalties and should avoid modifying penalties' init